backend/app/domain/helpers/email_templates/task_proposal_update.txt

```diff
@@ -0,0 +1,9 @@
+Greetings,
+
+Someone create the task: $name with the code: $code.
```

Comment:
Is this supposed to be sent to the task creator? If that's the case I suggest:

Greetings,

You're task $name with the code $code has been successfully created  with the following description: $desc.

Thanks,

The Dynabench Team